core/src/test/scala/integration/kafka/api/AuthorizerIntegrationTest.scala

```diff
@@ -491,6 +493,37 @@ class AuthorizerIntegrationTest extends BaseRequestTest {
     }
   }
 
+  @Test
+  def testCreateTopicAuthorizationWithClusterCreate() {
+    val requestKeyToRequest = mutable.LinkedHashMap[ApiKeys, AbstractRequest](
+      ApiKeys.CREATE_TOPICS -> createTopicsRequest
+    )
+
+    val requestKeysToAcls = Map[ApiKeys, Map[Resource, Set[Acl]]](
+      ApiKeys.CREATE_TOPICS -> clusterCreateAcl,
+    )
```

Comment:
It's a bit odd that we are using maps here even though there's a single entry in each. Shall we simplify?

Reply:
agreed!